lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -167,13 +176,49 @@
                 }
             ];
 
+            ordersHistoryData.unshift(
+                {
+                    'title': gettext('ORDER NAME'),
+                    'order_date': gettext('ORDER PLACED'),
+                    'price': gettext('TOTAL'),
+                    'number': gettext('INVOICE ID')
+                }
+            );
+
+            ordersSectionData = [
+                {
+                    title: gettext('Your Orders'),
+                    subtitle: gettext(
+                        'View details of our past purchases and, potentially request a refund for a past order. For ' +
```

Comment:
This page contains information about orders that you have placed with edX. 

OR

This page contains information about orders that you have placed.